src/Symfony/Component/Console/Command/DumpCompletionCommand.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Command;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\ConsoleOutputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Process\Process;
+
+/**
+ * Dumps the completion script for the current shell.
+ *
+ * @author Wouter de Jong <wouter@wouterj.nl>
+ */
+final class DumpCompletionCommand extends Command
+{
+    protected static $defaultName = 'completion';
+    protected static $defaultDescription = 'Dump the shell completion script';
+
+    protected function configure()
+    {
+        $fullCommand = $_SERVER['PHP_SELF'];
+        $commandName = basename($fullCommand);
+        $fullCommand = realpath($fullCommand) ?: $fullCommand;
+        $shell = self::guessShell();
+
+        $this
+            ->setHelp(<<<EOH
+The <info>%command.name%</info> command dumps the shell completion script required
```

Comment:
Small nitpick: we are using the short closing tags for internal commands:

```suggestion
The <info>%command.name%</> command dumps the shell completion script required
```

Can you fix all of them?